backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -377,12 +377,10 @@ public Parcel PendingUpdate(Parcel parcel)
                     var updateBuilding = parcel.Buildings.FirstOrDefault(pb => pb.BuildingId == parcelBuilding.BuildingId);
                     if (updateBuilding == null)
                     {
-                        // TODO: This will delete the building from the datasource.  This isn't entirely ideal as it may be related to multiple parcels.
                         this.ThrowIfNotAllowedToUpdate(parcelBuilding.Building, _options.Project);
 
                         var parcelBuildings = this.Context.ParcelBuildings.Where(pb => pb.BuildingId == parcelBuilding.BuildingId).ToArray();
                         parcelBuildings.ForEach(pb => this.Context.ParcelBuildings.Remove(pb)); // Remove all relationships from other parcels to this building.
-                        this.Context.Buildings.Remove(parcelBuilding.Building);
```

Comment:
In my opinion this no longer is required, as previously we needed to deleted buildings that were removed from parcels to prevent "orphaned" buildings.